Firestore/core/src/local/local_store.cc

```diff
@@ -207,7 +207,8 @@ LocalWriteResult LocalStore::WriteLocally(std::vector<Mutation>&& mutations) {
     for (const Mutation& mutation : mutations) {
       auto it = overlayed_documents.find(mutation.key());
       HARD_ASSERT(it != overlayed_documents.end(),
-                  "Failed to find overlayed document with a mutation key");
+                  "Failed to find overlayed document with mutation key: ",
```

Comment:
nit: Add `%s` to the assertion message so that the string will actually be included (I think HARD_ASSERT takes a printf-style formatting string, but I could be wrong about that).

Reply:
Fixed.